static/process-asm.ts

```diff
@@ -0,0 +1,548 @@
+import {ParseFiltersAndOutputOptions} from '../types/features/filters.interfaces.js';
```

Comment:
please add a comment to indicate where this file comes from. Is it copied from somewhere in the backend?

Reply:
Addressed